File: test/fixtures/java/gradle-demo/src/test/java/dev/hellmacs/demo/GreeterTest.java
package dev.hellmacs.demo;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

class GreeterTest {
    @Test
    void greetsByName() {
        assertEquals("Hello, Ann, from Hellmacs!", new Greeter("Hellmacs").greet("Ann"));
    }

    @Test
    void lombokGettersWork() {
        assertEquals(42, new Person("Doomguy", 42).getAge());
    }
}

File: test/fixtures/kotlin/gradle-demo/src/test/kotlin/dev/hellmacs/demo/GreeterTest.kt
package dev.hellmacs.demo

import kotlin.test.Test
import kotlin.test.assertEquals

class GreeterTest {
    @Test
    fun greetsByName() {
        assertEquals("Hello, Ann, from Hellmacs!", Greeter("Hellmacs").greet("Ann"))
    }

    @Test
    fun `greets someone with a backticked name`() {
        assertEquals("Hello, Bob, from Hellmacs!", Greeter("Hellmacs").greet("Bob"))
    }
}
